core/templates/dev/head/domain/state/StateObjectFactory.ts

```diff
@@ -77,7 +77,7 @@ angular.module('oppia').factory('StateObjectFactory', [
     /* eslint-disable dot-notation */
     State['createDefaultState'] = function(newStateName) {
     /* eslint-enable dot-notation */
-      var newStateTemplate = angular.copy(constants.NEW_STATE_TEMPLATE);
+      var newStateTemplate = NEW_STATE_TEMPLATE;
```

Comment:
Any reason to drop the angular.copy()? I'm fine with doing that if NEW_STATE_TEMPLATE is protected from modification even if something here modifies newStateTemplate, but let's keep the copy() otherwise.